src/dataprep/main/fields/fields_children.Rmd

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
how important is this file? can we delete it? or can we put parts of it elsewhere, for instance in `reports/quality_linking.Rmd` on the branch for #11 ?

Reply:
yes we can drop this/keep it somewhere else. I'll commit its deletion and think about where to put it.

Reply:
done